jekyll/_cci2/configuration-reference.md

```diff
@@ -804,17 +807,24 @@ docker_layer_caching | N | Boolean | Set this to `true` to enable [Docker Layer
 
 Example:
 
-```yaml
-version: 2.1
+{:.tab.machine.Cloud}
+```yml
 jobs:
-  build:
-    machine:
-      image: ubuntu-2004:202010-01
+  build: # name of your job
+    machine: # executor type
+      image: ubuntu-2004:202010-01 # # recommended linux image - includes Ubuntu 20.04, docker 19.03.13, docker-compose 1.27.4
```

Comment:
A couple of questions:

1. Where is there two hashes. I'm assuming a type?
2. This image is very old. From 2020. It will have older software and it's also using the old versioning scheme which might confuse users to thinking that's what they should use. We should use a newer image tag or even the `current` tag. I'm not sure what the Images team is currently recommending, @JalexChen?

Reply:
Oh I just copied this over from the using-linuxvm page, I guess it got outdated. And yes # # is probably a typo I'll remove. I'll wait on @JalexChen and then get them both updated